src/git.zsh

```diff
@@ -60,19 +60,36 @@ _fzf_complete_git() {
         resolved_commands+=($subcommand)
     done
 
-    local last_argument=${${(Qz)arguments}[-1]}
+    local last_argument=${${(Q)${(z)arguments}}[-1]}
+
+    if [[ ${(Q)${(z)arguments}} =~ '^git (checkout|log|rebase|reset)' ]]; then
+        if [[ ${${(Q)${(z)arguments}}[(r)--]} = -- ]]; then
+            if [[ ${(Q)${(z)arguments}} = 'git checkout '* ]]; then
+                _fzf_complete_git-unstaged-files '--untracked-files=no' "--multi $_fzf_complete_preview_git_diff $FZF_DEFAULT_OPTS" $@
+                return
+            fi
+
+            if [[ ${(Q)${(z)arguments}} =~ '^git (log|rebase)' ]]; then
+                _fzf_path_completion "$prefix" $@
```

Comment:
It is not appropriate passing all files to `git log` and` git rebase`. I think these should be passed the result of `git ls-files`. Can you create new function?